paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadata.kt

```diff
@@ -48,6 +50,7 @@ internal data class PaymentMethodMetadata(
     val paymentMethodSaveConsentBehavior: PaymentMethodSaveConsentBehavior,
     val linkMode: LinkMode?,
     val financialConnectionsAvailable: Boolean = DefaultIsFinancialConnectionsAvailable(),
+    val cardBrandFilter: CardBrandFilter = DefaultCardBrandFilter,
```

Comment:
I don't think we want to supply a default here. We have a factory about all of our tests go through, which should make it less annoying.

Reply:
Removed!